src/compiler/helpers.js

```diff
@@ -41,6 +42,12 @@ export function addHandler (
   modifiers: ?ASTModifiers,
   important: ?boolean
 ) {
+  // warn prevent and passive modifier
+  if (process.env.NODE_ENV !== 'production' && modifiers && modifiers.prevent && modifiers.passive) {
+    warn(
+      'prevent and passive are mutual exclusive.'
```

Comment:
hmmm, what about `passive and prevent can't be used together. Passive handler can't prevent default  event.`

It uses simpler word and explains the reason.